descriptor.md

```diff
@@ -137,11 +140,17 @@ If a useful algorithm is not included in the above table, it SHOULD be submitted
 [SHA-256][rfc4634-s4.1] is a collision-resistant hash function, chosen for ubiquity, reasonable size and secure characteristics.
 Implementations MUST implement SHA-256 digest verification for use in descriptors.
 
+When the _algorithm identifier_ is `sha256`, the _encoded_ portion MUST match `/[a-f0-9]{64}/`.
+Note that `[A-F]` MUST NOT be used here.
```

Comment:
I'd rather stay DRY by defining a hex encoding with these restrictions, and then saying that `sha256` and `sha512` MUST be hex-encoded.